package NDnnProto;


message THindmarshRoseConst {
    optional double A = 1;
    optional double B = 2;
    optional double C = 3;
    optional double D = 4;
    optional double R = 5;
    optional double S = 6;
    optional double XR = 7;
    optional double SpikingVariable = 8;
    optional double BurstingVariable = 9;
}

message THindmarshRoseState {
}

